receiptui/src/main/res/values/strings.xml

```diff
@@ -117,7 +117,9 @@
     <string name="transfer_to_program_account">Transfer to Program</string>
     <string name="transfer_to_user">Transfer to User</string>
     <string name="transfer_to_paypal_account">PayPal</string>
-    <string name="transfer_to_bank_card">Debit Card</string>
+    <string name="transfer_to_bank_card">Transfer to Debit /transfer_to_debit_card Credit Card</string>
```

Comment:
check the string Transfer to Debit /transfer_to_debit_card Credit Card is correct

Reply:
Resolved